executor/common_linux.h

```diff
@@ -4164,7 +4164,10 @@ static int do_sandbox_none(void)
 #if SYZ_EXECUTOR || SYZ_WIFI
 	initialize_wifi_devices();
 #endif
+#ifndef __ANDROID__
```

Comment:
What is supposed to define this? I don't think we define this anywhere.

Reply:
IIUC it is defined by the Android toolchain, and there are two other places where this ifdef is used.
I am not completely sure however that we're using the Android toolchain to build executor.

Reply:
We don't have Android toolchain anywhere on syzbot.

Reply:
Hmm, then those other ifdefs have no effect either.

Reply:
The one in flatbuffers is third-party and should be fine, includes on android may be slightly different I guess.

The one in common_linux.h also looks different and fine. It does not detect android kernel, but again what's present in headers. If somebody builds with android toolchain, then I guess that's required for successful build.

Reply:
Yeah, this makes sense.
In the meantime I figured out what was broken on Android, so we don't need to disable mounting.